src/sidebar/helpers/test/mention-suggestions-test.js

```diff
@@ -67,6 +71,95 @@ describe('combineUsersForMentions', () => {
       },
     );
   });
+
+  [
+    {
+      mentionMode: 'username',
+      expectedUsers: [
+        {
+          userid: 'acct:accent@example.com',
+          username: 'accent',
+          displayName: 'á',
+        },
+        {
+          userid: 'acct:cecilia1@example.com',
+          username: 'cecilia1',
+          displayName: 'Cecelia Davenport',
+        },
+        {
+          userid: 'acct:cecilia92@example.com',
+          username: 'cecilia92',
+          displayName: 'Cecelia Davenport',
+        },
+        {
+          userid: 'acct:no_accent@example.com',
+          username: 'no_accent',
+          displayName: 'a',
+        },
+      ],
+    },
+    {
+      mentionMode: 'display-name',
+      expectedUsers: [
+        {
+          userid: 'acct:accent@example.com',
+          username: 'accent',
+          displayName: 'á',
+        },
+        {
+          userid: 'acct:no_accent@example.com',
+          username: 'no_accent',
+          displayName: 'a',
+        },
```

Comment:
This test fails and expects these two items in the opposite order if `sensitivity:'base'` is not passed.